### PR TITLE
sql dump for database schema, no data

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: AgniSchema.sql [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+--
+-- Table structure for table `GroupChats`
+--
+
+DROP TABLE IF EXISTS `GroupChats`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `GroupChats` (
+  `name` varchar(30) NOT NULL,
+  `owner` int(10) unsigned NOT NULL,
```
Comment (jeinglis) [inline AgniSchema.sql]: I think this should use 'User id' as a foreign key
maybe change the attribute name to ownerId?

File: AgniSchema.sql [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+--
+-- Table structure for table `Messages`
+--
+
+DROP TABLE IF EXISTS `Messages`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `Messages` (
+  `messageId` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `senderId` int(10) unsigned NOT NULL,
```
Comment (jeinglis) [inline AgniSchema.sql:131]: should use user ID as a foreign key

File: AgniSchema.sql [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  `sentAt` datetime NOT NULL,
+  `content` text NOT NULL,
+  `groupName` varchar(30) NOT NULL,
+  PRIMARY KEY (`messageId`),
```
Comment (jeinglis) [inline AgniSchema.sql:135]: weren't we going to make this weak?

File: AgniSchema.sql [hunk 4]
```diff
@@ -0,0 +1,182 @@
+-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
+--
```
Comment (jeinglis) [inline AgniSchema.sql:2]: Looks good! Just a few corrections, and I think we should discuss how we want the Message entity to be structured.

File: AgniSchema.sql (hunk 3 above)
Comment (ExtraGravity) [inline AgniSchema.sql:135]: How do I make it weak?

File: AgniSchema.sql (hunk 4 above)
Comment (ExtraGravity) [inline AgniSchema.sql:2]: You don't like how it's currently structured?

File: AgniSchema.sql (hunk 2 above)
Comment (ExtraGravity) [inline AgniSchema.sql:131]: Added, the change is below so this will not become an "outdated diff"

**Pull request merged**